float64 jnt_0
float64 jnt_1
float64 jnt_2
float64 jnt_3
float64 jnt_4
float64 jnt_5
float64 jnt_6




